### DISCUSSION TITLE
Is there a way to use sigh to download all the profiles?

### DISCUSSION THREAD

**Discussion by tomvisions** [fastlane/fastlane#18258] [Q&A]
Hello, I read in the documentation a way to download all the profiles using sigh via the command line.

fastlane sigh download_all

However, this requires a login. I'm using the API to authenticate, so using sigh this way

sigh(api_key: api_key)

I've tried sigh(api_key: api_key, download_all: true) but sadly that doesn't appear to be arguments.

Any thoughts?

Thanks  for your time.

**Comment by tomvisions** [comment]
Found out a way to do it using the API.

**Reply by cjwhitsitt** [reply]
Could you explain how you did this via the API?
